src/features/favorites/screen/index.tsx

```diff
@@ -12,7 +12,7 @@ import FavoriteItem from './favoriteItem';
 import styles from './styles';
 
 const FavoritesScreen = () => {
-  const { data } = useGetFavorites();
+  const { data: { data: favorites = [] } = {} } = useGetFavorites();
```

Comment:
You should consider returning the direct value from the query, which will save you the destructuring and the default value here.

Reply:
I destructured this on purpose for cleanlines of code

Reply:
I'll leave it here as a reference, the thing is that `useGetFavorite`, should return something like:
`favorites: data?.data || []`

That would be cleaner because in the component you will not have any default value.